syft/serde.py

```diff
@@ -40,6 +40,10 @@
 import io
 import numpy
 import zstd
+import syft
```

Comment:
Is this import really needed?

Reply:
Yeah because I want the local_worker and thus call `syft.local_worker.get_worker(val_str, fail_hard=True)`